File: FantasyFishing/Models/Fish.cs
using System;
namespace FantasyFishing.Models
{
    public partial class Fish
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public object Img_Src_Set { get; set; }
        public Meta Meta { get; set; }
    }

    public partial class Meta
    {
        //public string Conservation_Status { get; set; }
        public object Scientific_Classification { get; set; }
        //public string Binomial_Name { get; set; }
        //public string Type_Species { get; set; }
        //public string Synonyms { get; set; }
        //public string Subfamilies_Genera { get; set; }
        //public string Genera { get; set; }
        //public string Families { get; set; }
        //public string Subfamily { get; set; }
        //public string Subfamilies { get; set; }
        //public string Species { get; set; }
    }
}

File: FantasyFishing/ClientApp/src/app/Models/fish.ts
export interface Fish {
    id: number;
    name: string;
    url: string;
    img_Src_Set: ImgLink;
    meta: MetaData;
}

export interface MetaData {
    
    scientific_Classification: SciClass;
    binomial_Name: string;
    type_Species: string;
    synonyms: string;
    subFamilies_Genera: string;
    genera: string;
    familes: string;
    subfamily: string;
    subfamilies: string;
    species: string;
}

export interface ImgLink {
    link1: string;
    link2: string;
}

export interface SciClass{
    kingdom: string;
    phylum: string;
    class: string;
    order: string;
    family: string;
    genus: string;
    species: string;
    subOrder: string;
    subFamily: string;
    infraClass: string;
    clade: string;
}
